manifests/t/the-sz/Yale/1.21/the-sz.Yale.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.Yale
+PackageVersion: 1.21
+PackageLocale: en-US
+Publisher: the sz development
+PackageName: Yale
+PackageUrl: https://the-sz.com/products/yale/
+License: Copyright (c) the-sz.com
+LicenseUrl: https://the-sz.com/products/license.php
+ShortDescription: Bring back the Classic Windows XP Start Menu to Windows 10. With one click you have access to all of your Start Menu entries. Plus direct access to the Control Panel, Administrator command line and Run dialog.
+Moniker: yale
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://the-sz.com/services/winget/get.php?p=yale

```

Comment:
```suggestion
  InstallerUrl: https://the-sz.com/services/winget/Setup.exe
```

Reply:
to distinguish between different products I would like to keep the php download script in between